pytorch_lightning/utilities/distributed.py

```diff
@@ -378,7 +378,8 @@ def init_dist_connection(
         )
 
 
-def _collect_states_on_rank_zero(state: Dict[str, Any], device: torch.device) -> Optional[Dict[int, Any]]:
+# TODO: Refactor with the Strategy Collectives once finalized.
+def _collect_states_on_rank_zero(state: Dict[str, Any]) -> Optional[Dict[int, Any]]:
```

Comment:
I believe this cannot return None anymore?

Reply:
Done !